apiVersion: aadpodidentity.k8s.io/v1
kind: AzurePodIdentityException
metadata:
  labels:
    app.giantswarm.io/branch: '{{ .Values.project.branch }}'
    app.giantswarm.io/commit: '{{ .Values.project.commit }}'
    app.kubernetes.io/instance: '{{ .Release.Name }}'
    app.kubernetes.io/managed-by: '{{ .Release.Service }}'
    app.kubernetes.io/name: '{{ template "aad-pod-identity.name" . }}'
    app.kubernetes.io/version: '{{ .Chart.AppVersion }}'
    helm.sh/chart: '{{ .Chart.Name }}'
  name: aks-addon-exception
  namespace: '{{ .Release.Namespace }}'
spec:
  podLabels:
    kubernetes.azure.com/managedby: aks
  template:
    metadata:
      labels:
        app.kubernetes.io/instance: '{{ .Release.Name }}'
        app.kubernetes.io/name: '{{ template "aad-pod-identity.name" . }}'
